student-cvs/Carolina08001-6984/contact.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact me</title>
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://kit.fontawesome.com/c2447c2ee1.js" crossorigin="anonymous"></script>
+
+</head>
+  
+<body>
+  <header id="main-header">
+    <nav>
+      <ul>
+        <li><a href="./index.html">Home</a></li>
+        <li><a href="./index.html/#bio">Biography</a></li>
+        <li><a href="./index.html/#certifications">Certifications completed</a></li>
+        <li><a href="./contact.html">Contact</a> </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="container">
+    <h1>Contacto</h1>
+    <form action="/server" method="get">
+      <div class="form-group">
+        <label for="name">Name</label>
+        <input id="name" type="text" name="nombres"
+        placeholder="Enter your name" required>
+      </div>
+      <div class="form-group">
+        <label for="email">Email</label>
+        <input id="email" type="email" name="email"
+        placeholder="email@email.com" required>
+      </div>
+      <div class="form-group">
+        <label for="telefono">Your phone </label>
+        <input type="tel" name="telefono" id="telefono" placeholder="Enter yor phone number"
+        required>
+      </div>
+      <div class="form-group">
+        <fieldset>
+          <legend>Budget</legend>
+          <label for="10000">
+          <input type="radio" name="budget" id="10000" value="10000">
+          $ 10,000.00</label>
+
+          <label for="5000"><input type="radio" name="budget" id="5000" value="5000">
+          $ 5,000.00</label>
+
+          <label for="3000"><input type="radio" name="budget" id="3000" value="3000">
+          $ 3,000.00</label>
+
+          <label for="3000"><input type="radio" name="budget" id="1000" value="1000">
+          $ 1,000.00</label>
```

Comment:
Hay que corregir el `for`
```suggestion
          <label for="1000"><input type="radio" name="budget" id="1000" value="1000">
          $ 1,000.00</label>
```